RecoTracker/SiTrackerMRHTools/plugins/MultiRecHitCollectorESProducer.cc

```diff
@@ -1,73 +1,91 @@
+// user includes
+#include "FWCore/Framework/interface/ESHandle.h"
```

Comment:
Is this still needed?

Reply:
removed in [0a9ee2d](https://github.com/cms-sw/cmssw/pull/36980/commits/0a9ee2def74a0cf443e5e22a5b52be489d5a03de)